gitlab/v4/objects/merge_requests.py

```diff
@@ -165,9 +165,7 @@ class ProjectMergeRequest(
 
     @cli.register_custom_action("ProjectMergeRequest")
     @exc.on_http_error(exc.GitlabMROnBuildSuccessError)
-    def cancel_merge_when_pipeline_succeeds(
-        self, **kwargs: Any
-    ) -> "ProjectMergeRequest":
+    def cancel_merge_when_pipeline_succeeds(self, **kwargs: Any) -> Dict[str, str]:
```

Comment:
We just need a corresponding docstring change here (we have `Returns: ProjectMergeRequest` atm) (sorry can't comment on unchanged lines)

Reply:
Done